assets/scripts/store/reducers/__tests__/street.test.js

```diff
@@ -222,6 +222,35 @@ describe('street reducer', () => {
     })
   })
 
+  describe('CLEAR_LOCATION', () => {
+    it('sets street.location to be null and street name to DEFAULT_NAME if userUpdated = False', () => {
```

Comment:
For the purposes of the test, it should be fine to mock out whatever we need to for this. Eventually `DEFAULT_NAME` has to be deprecated because we can't hard-code the string anymore (or we set `DEFAULT_NAME` via the same `t()` function`.

For `userUpdated = False`, for clarity, either use JS notation (`userUpdated === false`) or plain English (userUpdated is \`false\`).